docs/fastdds/xtypes/language_binding.rst

```diff
@@ -0,0 +1,1000 @@
+If the position is not specified, sequential order is followed.
+
+The |DynamicTypeBuilderFactory-api| exposes the function |DynamicTypeBuilderFactory::create_bitmask_type| to
+facilitate the creation of bitmask types.
+
+Bitmask types can be manipulated either using the |DynamicData::get_boolean_value|/|DynamicData::set_boolean_value| in
+order to set a specific bitflag, or by using the unsigned integer setter/getter corresponding to the bitmask bound.
+In this latest case, only bitflags are going to be set (bits not named are always unset).
+
+.. tabs::
+
+    .. tab:: IDL
+
+        .. literalinclude:: /../code/DynamicTypesIDLExamples.idl
+            :language: idl
+            :start-after: //!--IDL_BITMASK
+            :end-before: //!--
+
+    .. tab:: XML
+
+        .. literalinclude:: /../code/XMLTester.xml
+            :language: xml
+            :start-after: <!-->XML_BITMASK<-->
+            :end-before: <!--><-->
+
+    .. tab:: C++
+
+        .. literalinclude:: /../code/DDSCodeTester.cpp
+            :language: c++
+            :start-after: //!--CPP_BITMASK
+            :end-before: //!--
+
+For a detailed explanation about the XML definition of this type, please refer to
+:ref:`XML Bitmask Types<xmldynamictypes_bitmask>`.
+
+
+.. _xtypes_supportedtypes_alias:
+
+Alias Types
+^^^^^^^^^^^
+
+Alias types provide an alternative name to an already existing type.
+The :code:`TypeKind` used to identify aliases is :code:`TK_ALIAS`.
+Besides defining the alias name, the underlying type must be set using |TypeDescriptor-api| :code:`base_type` property.
+Alias recursion is supported by defining another alias type as the base type.
+
+Once the |DynamicData-api| is created, information can be accessed as if working with the base type.
+
+.. tabs::
+
+    .. tab:: IDL
+
+        .. literalinclude:: /../code/DynamicTypesIDLExamples.idl
+            :language: idl
+            :start-after: //!--IDL_TYPEDEF
+            :end-before: //!--
+
+    .. tab:: XML
+
+        .. literalinclude:: /../code/XMLTester.xml
+            :language: xml
+            :start-after: <!-->XML_TYPEDEF<-->
+            :end-before: <!--><-->
+
+    .. tab:: C++
+
+        .. literalinclude:: /../code/DDSCodeTester.cpp
+            :language: c++
+            :start-after: //!--CPP_TYPEDEF
+            :end-before: //!--
+
+For a detailed explanation about the XML definition of this type, please refer to
+:ref:`XML Alias Types<xmldynamictypes_typedef>`.
+
+.. _xtypes_supportedtypes_sequence:
+
+Sequence types
+^^^^^^^^^^^^^^
+
+Sequence types are one-dimensional collections of any type.
+The :code:`TypeKind` used to identify sequences is :code:`TK_SEQUENCE`.
+|TypeDescriptor-api| :code:`element_type` property must be set with the collection's type.
+Additionally, :code:`bound` property might be configured to set a maximum length for the collection.
+Otherwise, the collection is considered to be unbounded.
```

Comment:
This is not true. It is mandatory to set the bound. For unbounded `LENGTH_UNLIMITED` must be set.